TestCases/tutorials.py

```diff
@@ -324,6 +344,22 @@ def main():
 
     pass_list = [ test.run_test() for test in test_list ]
 
+
+    # design-FADO: 90 degree pipe bend optimization 
+    sudo_design_fado = TestCase('sudo_bend_design_fado')
+    sudo_design_fado.command  = TestCase.Command(exec = "python", param = "optimization.py")
+    sudo_design_fado.cfg_dir = "../Tutorials/design/Inc_Turbulent_Bend_Wallfunctions"
+    sudo_design_fado.cfg_file = "sudo.cfg"
+    sudo_design_fado.multizone = False
+    sudo_design_fado.test_iter = 10
+    sudo_design_fado.timeout = 1600
+    sudo_design_fado.reference_file   = "optim.csv.ref"
```

Comment:
Move this file to the code repo to make it easier to update when there are changes.